### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jameshiggie**
Teamcity pytests pass on current, past and next Seeq versions

Manual test install and functional tests pass on R60 server

### CODE REVIEW

File: docs/_sources/changelog.md.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+v0.3.5
+---
+Dtw improvments, fucntions to package MPS for use in addon manager
```
Comment (Flaminator89) [inline docs/_sources/changelog.md.txt:5]: functions typo

File: docs_src/changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+v0.3.5
+---
+Dtw improvments, fucntions to package MPS for use in addon manager
```
Comment (Flaminator89) [inline docs_src/changelog.md:5]: functions typo

File: seeq/addons/mps/_version.py [hunk 3]
```diff
@@ -1 +1 @@
-__version__ = '0.3.4'
\ No newline at end of file
+__version__ = '0.3.6'
```
Comment (Flaminator89) [inline seeq/addons/mps/_version.py:1]: Should this be 0.3.5?

File: build.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        directory = pathlib.Path("./additional_content/")
+        for file in directory.iterdir():
+            z.write(file)
+        addon_manager_artifacts.append(addon_meta)
+
+    print('Successfully created.')
+
+if args.distribute:
+
+    # THIS BLOCK OF CODE IS NO MORE IN USE BUT COULD LATER BE USED IN FUTURE PURPOSES
```
Comment (Flaminator89) [inline build.py:112]: If its not being used now - prob best to just remove it.

**Review by Flaminator89 — APPROVED** [review]
Approved once we clarify the 0.3.6 version reference.

File: seeq/addons/mps/_mps_sdl_ui.py [hunk 5]
```diff
@@ -30,6 +30,9 @@ def __init__(self, server, workbook_id, worksheet_id):
         self.workbook_id = workbook_id
         self.worksheet_id = worksheet_id
 
+        # Set the compatibility option so that you maximize the chance that SPy will remain compatible with your notebook/script
+        spy.options.compatibility = 187.5
```
Comment (anvenkat18) [inline seeq/addons/mps/_mps_sdl_ui.py:34]: Will mps break with spy 188.0 or 188.3?

**Review by anvenkat18 — APPROVED** [review]
Did not test.

Installation in the docs will need to be updated to show how to install/upgrade with add-on manager when the latter is close to release.

**Pull request merged**